src/pages/CommandView/CommandView.tsx

```diff
@@ -51,7 +51,7 @@ const CommandView = () => {
     ? getJobSchema(getSchema(instances, parameters))
     : getSchema(instances, parameters)
   const uiSchema = getUiSchema(instances, parameters)
-  const model = getModel(parameters)
+  const model = getModel(parameters, theSystem.instances, isJob)
```

Comment:
Previously, the model was determined by the form when it parsed out defaults. That was a silly idea, so now the model is determined from the correct data.